### ISSUE TITLE
[Feature Request]: [SF_ACC] [Carousel] Keyboard users cannot focus on or interact with navigation buttons on the carousel

### ISSUE THREAD

**Issue by bonnieso** [SAP/ui5-webcomponents#10693]
### Feature Request Description

Currently, the carousel component is only navigable via the left and right arrow keys for keyboard users. Users are not able to tab to focus on the carousel navigation buttons and additionally the buttons are not interactive. This causes confusion for keyboard users because visually there are buttons on the carousel which they expect to be interactive elements.

### Proposed Solution

It would be a better experience for users if they could tab to the navigation buttons and trigger the them via the "Space" or "Enter" key like any other button element.

### Organization

SAP SuccessFactors

### Priority

Medium

### Privacy Policy

- [x] I’m not disclosing any internal or sensitive information.

**Comment by olannyv** [comment]
@bonnieso 
This is working according to the design specification. 
I will rise this new feature request internally and will come back after it is approved or rejected.

BR, Daniel

**Comment by olannyv** [comment]
An clarification Jira ticket -453 is created with our SAP DESIGN SYSTEM team.
@bonnieso please follwo-up in the Jira or with me (Daniel Vladinov) via MS Teams@SAP
